server/containers/README.md

```diff
@@ -0,0 +1,123 @@
+# Kaa Docker image
+
+This readme contains information about Kaa Docker image deployment.
+
+Default environments that comes in this package is:
+  - Fully functional Kaa cluster (Nx Kaa node, 1x Zookeeper node, 1x Database SQL node, 1x Database NoSQL node)
+    - Kaa node
+    - Zookeeper node
+    - MariaDb/PostgreSQL
+    - MongoDB/Cassandra
+
+Base image configuration is done using the following environment variables:
+
+| VARIABLE         		       	|   DEFAULT					| NOTE / POSSIBLE VALUES
+| -----------------------------	|--------------------------	| ----------------------------
+| SERVICES_WAIT_TIMEOUT			| -1 (forever)				| Seconds (integer) before timeout while waiting for ZK/SQL/NoSQL to be ready, otherwise abort.<br>10: wait 10 seconds.<br>0: don't wait<br>-1: wait forever.
+|								|							|
+| ZOOKEEPER_NODE_LIST			| localhost:2181			| <i>comma separated list</i>
+| 								| 							|
+| SQL_PROVIDER_NAME				| mariadb 					| mariadb , postgresql
+| JDBC_HOST						| localhost					|
+| JDBC_PORT						| if mariadb: 3306<br>if postgresql: 5432|
+| JDBC_USERNAME					| sqladmin					|
+| JDBC_PASSWORD					| admin						|
+| JDBC_DB_NAME					| kaa 						|
+ 								| 							|
+| CASSANDRA_CLUSTER_NAME		| Kaa Cluster 				|
+| CASSANDRA_KEYSPACE_NAME		| kaa 						|
+| CASSANDRA_NODE_LIST			| localhost:9042 			| <i>comma separated list</i>
+| CASSANDRA_USE_SSL				| false 					|
+| CASSANDRA_USE_JMX				| true 						|
+| CASSANDRA_USE_CREDENTIALS		| false 					|
+| CASSANDRA_USERNAME 			| (empty) 					|
+| CASSANDRA_PASSWORD 			| (empty) 					| 
+| 								| 							| 
+| MONGODB_NODE_LIST 			| localhost:27017 			| 
+| MONGODB_DB_NAME				| kaa 						| 
+| MONGODB_WRITE_CONCERN 		| acknowledged 				| 
+| 								| 							| 
+| NOSQL_PROVIDER_NAME			| mongodb 					| mongodb , cassandra
+|								|							|
+| CONTROL_SERVER_ENABLED		| true						| true/false
+| BOOTSTRAP_SERVER_ENABLED		| true						| true/false
+| OPERATIONS_SERVER_ENABLED		| true						| true/false
+| THRIFT_HOST					| localhost					| 
+| THRIFT_PORT					| 9090						| 
+| ADMIN_PORT					| 8080						| 
+| SUPPORT_UNENCRYPTED_CONNECTION| true						| true/false
+| TRANSPORT_BIND_INTERFACE		| 0.0.0.0					| 
+| TRANSPORT_PUBLIC_INTERFACE	| current public host					|
+| METRICS_ENABLED				| true 						| true/false
+ 
+## Steps to deploy
+
+1. Download debian package from [official site](http://www.kaaproject.org/download-kaa/)
```

Comment:
This step needs to be improved. I suggest to download deb package automatically from https://github.com/kaaproject/kaa/releases (navigate to particular release for more details. relese version should be a configuration parameter with default value equal to current release.